extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css

```diff
@@ -0,0 +1,243 @@
+}
+
+amp-story-player a {
+  display: none !important;
+}
+
+.i-amphtml-story-dev-tools-device-chips-container {
+  position: absolute !important;
+  bottom: 0 !important;
+  width: 100% !important;
+  text-align: center !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip {
+  border-radius: 16px !important;
+  background: #2e2e2e !important;
+  border: 1px solid #2e2e2e !important;
+  height: 32px !important;
+  font-size: 13px !important;
+  line-height: 20px !important;
+  display: inline-flex !important;
+  align-items: center !important;
+  padding: 0 6px 0 10px !important;
+  margin: 0 5px !important;
+  max-width: 300px !important;
+  transition: background 0.2s ease-in-out, color 0.2s ease-in-out, max-width 0.2s ease-in-out !important;
+  white-space: nowrap !important;
+  cursor: pointer !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip:hover {
+  background: #3d3d3d !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip svg {
+  cursor: pointer !important;
+  background: black !important;
+  border-radius: 50% !important;
+  padding: 2px !important;
+  margin-left: 10px !important;
+  transform: rotate(0deg) !important;
+  transition: transform 0.2s ease-in-out !important;
+}
+
+.i-amphtml-story-dev-tools-device-chips .i-amphtml-story-dev-tools-device-chip[inactive] {
+  max-width: 0px !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip[inactive] {
+  background: transparent !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip[inactive]:hover {
+  background: #1d1d1d !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip[inactive] svg {
+  transform: rotate(90deg) !important;
+  background: #ffffff22 !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip:not([inactive]):not(.i-amphtml-story-dev-tools-add-device) .i-amphtml-story-dev-tools-device-chip-add-stick {
+  fill: transparent !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device {
+  background: #79B3FF !important;
+  color: black !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device svg {
+  fill: #79B3FF !important;
+  background: black !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device:hover {
+  background: #92c1ff !important;
+}
+
+.i-amphtml-story-dev-tools-device-popup-bg {
+  position: fixed !important;
+  left: 0 !important;
+  top: 0 !important;
+  height: 100% !important;
+  width: 100% !important;
+  background: transparent !important;
+  display: flex !important;
+  justify-content: center !important;
+  align-items: center !important;
+  z-index: 1 !important;
+  transition: background 0.2s ease-in-out !important;
+}
+
+.i-amphtml-story-dev-tools-device-popup-container {
+  background: black !important;
+  width: min(650px, 90%) !important;
+  border-radius: 10px !important;
+  padding: 40px !important;
+  position: relative !important;
+  transition: transform 0.2s ease-in-out, opacity 0.2s ease-in-out !important;
```

Comment:
I hope @processprocess will yell at your for not using his cool timing functions :))

Reply:
Lol we were talking about this animation, would be nice if we used sth like `scale-fade-up` or similar. Didn't want to get into polishing much but open for recommendations!

Reply:
Let's use the default material curves:
- accelerating for when exiting/disappearing on the screen
- decelerating for when entering/appearing on the screen
- standard curve for in screen movement

https://material.io/design/motion/speed.html#easing